refresh/manager.go

```diff
@@ -54,8 +54,6 @@ func (r *Manager) Start() error {
 					if event.Op != fsnotify.Chmod {
```

Comment:
You should use `event.Has(fsnotify.Chmod)` here; in some (rare) cases there can be multiple events.

Unrelated to this PR really, but just something I noticed.

Reply:
Thanks for the comment. However, the answer is no.

The fancy method `Has()` was introduced from v1.6.0 of fsnotify and it should be used when we just want to check if the value has a specific bit, regardless of other bits. In that case, your suggestion is correct.

However, the logic is that the following `build()` should be prevented when only `CHMOD` happen without any events indicating a real change. For example, if the value is `CHMOD|WRITE`, which has `CHMOD` along with `WRITE`, we should run `build()` and check with `Has()` here is not correct in this case.

Reply:
I missed adding this code comment when I upgrade fsnotify to v1.6.0 to prevent possible future confusion.

Reply:
One quick question. Do you have any plan to support `HasNot()` or similar?

Reply:
CHMOD | WRITE could indicate there was a write and a chmod though; in that case I think you'll want to do a rebuild?

I don't think I've ever seen that happen though, so it's a really hypothetical issue and probably not worth worrying about one way or the other.

Reply:
Sorry, I didn't see your further comments.

> when I upgrade fsnotify to v1.6.0

You updated it in this PR.

> One quick question. Do you have any plan to support HasNot() or similar?

Not really; I think !Has() should be good enough? I try to keep the API as small as possible, as I usually find that easier to use (less documentation, less stuff in completion, etc.)

Reply:
Yes, Refresh rebuilds the supporting app in any of the other cases except CHMOD only. What is the possible case then? (CHMOD with which?)

Reply:
I saw one `CHMOD | WRITE` event when I was testing for the bug other day. So it can happen.

Reply:
On Linux chmod gets sent if a file is unlinked (removed) while there are still open file descriptors. There may be some other caveats too (for example kqueue used by macOS and BSD isn't very well documented on this).

It's actually pretty tricky to determine "did this file change?" reliably; the general strategy I've used is watch the parent directory of the file you want to watch, watch for Create and Write events, and debounce the events by 100ms to make sure the write is finished (there's an example of that in `./cmd/fsnotify`).`

That's the most reliable way I've figured out so far that works well on all platforms in all situations as far as I know. Things are better in v1.6.0 than they used to be in older versions though.

Reply:
> > One quick question. Do you have any plan to support HasNot() or similar?
> 
> Not really; I think !Has() should be good enough? I try to keep the API as small as possible, as I usually find that easier to use (less documentation, less stuff in completion, etc.)

Agree. `Has()` could be enough for most usages and `HasNot()` will not be popular I guess (so for the edge case like this, we can still use bit operation or just compare with a value). However, `!Has(A)` is not equal to `HasNot(A)`.

Reply:
if the CHMOD event can be delivered with the other, including write, remove, or something else, we should not filter them out so the current logic should check CHMOD-only event. If there is no `HasNot()` API, `!= CHMOD` is correct here.

Reply:
anyway, nice to e-meet you @arp242!

Reply:
> However, `!Has(A)` is not equal to `HasNot(A)`.

(Oh, this is not literally true but I mean in this context.)